content/chainguard/chainguard-images/features/cve_visualizations/index.md

```diff
@@ -13,59 +13,88 @@ tags: ["CONCEPTUAL", "CHAINGUARD IMAGES", "PRODUCT"]
 images: []
 menu:
   docs:
-    parent: "chainguard-images"
+    parent: "features"
 weight: 025
 toc: true
 ---
 
-CVE Visualizations provide insight into image health and can help teams measure the engineering,
-security, and economic benefits gained from using Chainguard Images. The feature creates reports
-with CVE comparisons between Chainguard Images and popular alternatives, as well as historical CVE
-remediation metrics.
+Chainguard provides CVE Visualizations for all of its images. This feature creates reports with CVE comparisons between Chainguard Images and popular alternatives, as well as historical CVE remediation metrics. CVE Visualizations provide insight into image health and can help teams measure the engineering, security, and economic benefits gained from using Chainguard Images
 
-> **NOTE**: This is a beta feature and changes should be expected. In particular, we plan to add more
-metrics in the near future.
+This guide outlines how you can access an image's CVE Visualization in both the Chainguard Console and in the Images Directory. 
 
-## Accessing CVE Visualizations
 
-Visualizations can be found under the [Reports](https://console.chainguard.dev/reports) section
-on the left-hand navigation bar.
+## Accessing CVE Visualizations in the Console
 
-The reports page should look similar to the following:
+You can find CVE Visualizations and reports in two places in the [Chainguard Console](https://console.chainguard.dev): in the **Reports** section of the left-hand navigation menu and in the **Comparison** tab of an individual Image's overview.
```

Comment:
```suggestion
You can find CVE Visualizations and reports two separate places in the [Chainguard Console](https://console.chainguard.dev): in the **Reports** section of the left-hand navigation menu and in the **Comparison** tab of an individual Image's overview.
```

Thought it may make sense to not have "in" so many times in the sentence